From 3cef0cc4ff17b6aab06007fcdcffbc61409f4c4b Mon Sep 17 00:00:00 2001
Message-ID: <3cef0cc4ff17b6aab06007fcdcffbc61409f4c4b.1718642985.git.git@grubix.eu>
In-Reply-To: <b33900237b28f5a8b943ca9048536d2cde19151b.1718642985.git.git@grubix.eu>
References: <b33900237b28f5a8b943ca9048536d2cde19151b.1718642985.git.git@grubix.eu>
From: Michael J Gruber <git@grubix.eu>
Date: Mon, 17 Jun 2024 18:16:32 +0200
Subject: [PATCH 2/2] Replace `delete-line` with its definition

37c022ae ("Use `without-restriction` in `with-temporary-notmuch-message-buffer`", 2024-03-14)
introduced `delete-line` in a test, but this is Emacs 29 and above only.
Replace it with its (almost) definition.
---
 test/T630-emacs-draft.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/T630-emacs-draft.sh b/test/T630-emacs-draft.sh
index 1fad58a8..7d0da21a 100755
--- a/test/T630-emacs-draft.sh
+++ b/test/T630-emacs-draft.sh
@@ -76,7 +76,7 @@ add_email_corpus attachment
 test_begin_subtest "Saving a draft keeps hidden headers"
 test_emacs '(notmuch-mua-new-reply "id:874llc2bkp.fsf@curie.anarc.at")
             (message-goto-subject)
-            (delete-line)
+            (delete-region (line-beginning-position) (line-beginning-position 2))
             (insert "Subject: draft-test-reply\n")
 	    (test-output "DRAFT")
 	    (notmuch-draft-postpone)
-- 
2.45.2.749.g54362de8d7

